examples/tutorials/advanced/Federated Recurrent Neural Network.ipynb

```diff
@@ -0,0 +1,985 @@
+{
```

Comment:
<p>&gt; Following distributed training, the resulting model is going to be able to perform operations like: ?</p><p><br></p><p>Missing some text here.</p>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2128/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>It was actually referring to the next cell's predictions. I added some more introductory text and a brief explanation about RNNs' training BTW.</p>